activesupport/lib/active_support/testing/time_helpers.rb

```diff
@@ -160,6 +160,15 @@ def travel_back
         simple_stubs.unstub_all!
       end
 
+      # Calls `travel_to` with `Time.now`.
+      #
+      #   freeze_time
+      #   Time.now # => 2017-07-05 11:01:38 +0530
+      #   Time.now # => 2017-07-05 11:01:38 +0530
+      def freeze_time
+        travel_to Time.now
```

Comment:
Shouldn't this be `Time.current` though, just so we can take `Time.zone` into account?

I wonder if it matters ...